examples/import_credentials.py

```diff
@@ -0,0 +1,28 @@
+'''
+As an alternative to exporting your credentials into the environment, 
+you may chose to create a separate file, next to your main file, 
+called credentials.py containing the following values:
```

Comment:
I don't think this is a method to recommend to handle secrets, because:

 - a quick accident could happen and this file would be committed and the secrets made public,
 - if the file is missing, continuous integration might not like it

In general, prefer something like dotenv https://pypi.org/project/python-dotenv

Reply:
As it's not linked to the file above, feel free to either remove the file, or to add the dotenv alternative somewhere in the doc. Personally I would let developers manage the secrets how they want it

Reply:
That makes sense. I will remove the file.

Reply:
Done.